spec/support/tag_resources_shared_context.rb

```diff
@@ -1,11 +1,12 @@
 RSpec.shared_context "tag_resource_objects" do
   let(:tenant1) { create(:tenant) }
   let(:tenant2) { create(:tenant, :external_tenant => 'fred') }
-  let(:workflow1) { create(:workflow, :tenant => tenant1, :sequence => 10) }
+  let(:tenant1_workflows) { create_list(:workflow, 2, :tenant => tenant1) }
+  let(:workflow1) { tenant1_workflows[1] }
   let(:wf1_tag)   { "/approval/workflows=#{workflow1.id}" }
-  let(:workflow2) { create(:workflow, :tenant => tenant1, :sequence => 5) }
+  let(:workflow2) { tenant1_workflows[0] }
   let(:wf2_tag)   { "/approval/workflows=#{workflow2.id}" }
-  let(:workflow3) { create(:workflow, :tenant => tenant2, :sequence => 7) }
+  let(:workflow3) { create(:workflow, :tenant => tenant2) }
```

Comment:
Were these just necessary changes due to the `before_validation` changes? I don't see any use of `workflow1/2/3` anywhere in files you touched so that's why I'm curious.

Reply:
Because of this PR, hard coded sequence number is not honored but depend on the order of workflow creations. So I introduced `tenant1_workflows` to ensure `workflow1` is before `workflow2`. `workflow/1/2/3` are used in https://github.com/RedHatInsights/approval-api/blob/master/spec/services/workflow_find_service_spec.rb#L45